blockchain/blockchain.go

```diff
@@ -1742,6 +1746,10 @@ type insertStats struct {
 // report prints statistics if some number of blocks have been processed
 // or more than a few seconds have passed since the last message.
 func (st *insertStats) report(chain []*types.Block, index int, cache common.StorageSize) {
+	// report will leave a log only if inserting two or more blocks at once
+	if len(chain) <= 1 {
+		return
+	}
```

Comment:
How about moving this logic into `insertChain` and adding one more if condition `if len(chain)<=1 && writeResult.Status == CanonStatTy `
https://github.com/klaytn/klaytn/blob/5dd354fedbc513e37de8c77c53a8755728dd406f/blockchain/blockchain.go#L1647

Reply:
Could you please explain the reason?

Reply:
I thought it is a little bit akward restricting the number of block to be reported inside of the method. It the method was intended to handling just one block, it should get a parameter `*types.Block` instead of `[]*types.Block`. 

`writeResult.Status == CanonStatTy` was just mentioned because `SideStatTy` doesn't not affected by this change. But, it is a very trivial comment.

Reply:
As the report decision logic already exists in the `report` function, I think it is okay to add another check logic in the function. Let's merge this change for now and change if it looks still awkward in the future.

Reply:
Um.. okay. It's not a big deal.